mcc/odin/odin/snapshot_list.go

```diff
@@ -0,0 +1,18 @@
+package odin
+
+import (
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
+)
+
+// ListSnapshots return a list of all DBSnapshots.
+func ListSnapshots(
+	instanceName string,
+	svc rdsiface.RDSAPI,
```

Comment:
`rdsiface.RDSAPI` is a mockable library interface for AWS RDS.
`svc` will be the object representing it.
It is being passed here, since `ListSnapshots` will need to interact with it to query for the corresponding instances.
In the future, it could be a global, or part of some other global structure, but right now, we're using it like this, in any other function related to a command.